README.md

```diff
@@ -1,2 +1,118 @@
-# strigo-github-exercises-examples
-Examples for markdown exercises that may be used in Strigo training events
+# Strigo GitHub Class Exercises examples
+
+## Quick start
+
+1. Create `.strigo/config.yml` file in your exercises' repository
+1. Configure exercises in `.strigo/config`
+
+   ```yaml
+   ---
+   classes:
+     - exercises:
+         # `file` is relative to .strigo directory
+         - file: "../Exercise.md"
+           title: "Title!"
+   ```
+
+1. In Strigo Class editor, fetch Exercises for a desired Class
+1. Done!
+
+---
+
+## Config file structure
+
+- `classes` - array of Classes
```

Comment:
In YAML, there's no "array", only "list".

Reply:
Also, array of Classes => A list of classes (required)

Reply:
They actually call it a "sequence" but there are lots of "array" mentions as well. I can live with a "list" :)

Reply:
Oh.. true.